DigitalLearningSolutions.Web/Views/ApplicationSelector/Index.cshtml

```diff
@@ -0,0 +1,93 @@
+        <div class="nhsuk-card__content">
+          <h2 class="nhsuk-card__heading nhsuk-heading-m">
+            <a class="nhsuk-card__link" asp-controller="LearningPortal" asp-action="Current">Learning Portal</a>
+          </h2>
+          <p class="nhsuk-card__description">Access to your current, available and completed learning courses.</p>
+        </div>
+      </div>
+    </div>
+  }
+  @if (Model.TrackingSystemAccess)
+  {
+    <div class="nhsuk-grid-column-one-half">
+      <div class="nhsuk-card nhsuk-card--clickable">
+        <div class="nhsuk-card__content">
+          <h2 class="nhsuk-card__heading nhsuk-heading-m">
+            <a class="nhsuk-card__link" href="@Configuration["CurrentSystemBaseUrl"]/tracking/dashboard">Tracking System</a>
+          </h2>
+          <p class="nhsuk-card__description">Manage and distribute learning to your organisation and access reports.</p>
+        </div>
+      </div>
+    </div>
+  }
+  @if (Model.ContentManagementSystemAccess)
+  {
+    <div class="nhsuk-grid-column-one-half">
+      <div class="nhsuk-card nhsuk-card--clickable">
+        <div class="nhsuk-card__content">
+          <h2 class="nhsuk-card__heading nhsuk-heading-m">
+            <a class="nhsuk-card__link" href="@Configuration["CurrentSystemBaseUrl"]/cms/courses">Content Management System</a>
+          </h2>
+          <p class="nhsuk-card__description">Import and manage learning content that's delivered through the Digital Learning Solutions platform.</p>
+        </div>
+      </div>
+    </div>
+  }
+  @if (Model.SuperviseAccess)
+  {
+    <div class="nhsuk-grid-column-one-half">
+      <div class="nhsuk-card nhsuk-card--clickable">
+        <div class="nhsuk-card__content">
+          <h2 class="nhsuk-card__heading nhsuk-heading-m">
+            <a class="nhsuk-card__link" href="@Configuration["CurrentSystemBaseUrl"]/tracking/supervise">Supervise</a>
+          </h2>
+          <p class="nhsuk-card__description">Assign and review staff competency self assessments and arrange supervision sessions.</p>
+        </div>
+      </div>
+    </div>
+  }
+  @if (Model.ContentCreatorAccess)
+  {
+    <div class="nhsuk-grid-column-one-half">
+      <div class="nhsuk-card nhsuk-card--clickable">
+        <div class="nhsuk-card__content">
+          <h2 class="nhsuk-card__heading nhsuk-heading-m">
+            <a class="nhsuk-card__link" href="@Configuration["CurrentSystemBaseUrl"]/cms/contentcreator">Content Creator</a>
+          </h2>
+          <p class="nhsuk-card__description">Create interactive elearning and assessments.</p>
+        </div>
+      </div>
+    </div>
+  }
+  @if (Model.FrameworksAccess)
+  {
+    <div class="nhsuk-grid-column-one-half">
+      <div class="nhsuk-card nhsuk-card--clickable">
+        <div class="nhsuk-card__content">
+          <h2 class="nhsuk-card__heading nhsuk-heading-m">
+            <a class="nhsuk-card__link" asp-controller="Frameworks" asp-action="MyFrameworks">Frameworks</a>

```

Comment:
`asp-action="MyFrameworks"` is used on other pages, and works, but visual studio complains that it can't resolve the action. Is there a fix for this or a better approach I should be using?

Reply:
I think it's because the controller action isn't actually named that, looks like it is FrameworksDashboard with a custom Route `[Route("/Frameworks/MyFrameworks/{page=1:int}")]` Visual Studio recognises it if I try changing it to that but I don't know that it is the correct link

Reply:
This is the correct link! 👍